app/src/main/res/values/strings.xml

```diff
@@ -59,6 +59,7 @@
     <string name="storage_fake_drive_summary">Needs to be plugged in</string>
     <string name="storage_available_bytes"><xliff:g example="1 GB" id="size">%1$s</xliff:g> free</string>
     <string name="storage_fake_nextcloud_title" translatable="false">Nextcloud</string>
+    <string name="not_recommended">Not recommended</string>
```

Comment:
can we not add the brackets here and simply concatenate the two strings?

Reply:
Regrettably, I'd probably be the last to know for sure whether this is the case, but it seems to me like either way of doing this may pose localization challenges.

What about something like:
`<string name="not_recommended_provider"><xliff:g example="Skynet">%1$s</xliff:g> (Not recommended)</string>`

...and plugging in whatever the provider is? This way, translators can figure out the proper way of wording the parenthetical and where it should go.

Reply:
Done.